docs/superpowers/plans/2026-05-25-github-pr-comments.md

```
@@ -0,0 +1,1940 @@
+
+- [ ] **Step 1: Update DESIGN.md Section 2**
+
+Add `GITHUB_ACTIONS` to the compact detection list and detection logic. In the env vars list (~line 418-428 equivalent in DESIGN.md), add the bullet:
+
+```markdown
+- GITHUB_ACTIONS: GitHub Actions CI
+```
+
+And update the detection logic block:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)